src/js/jwt/jwt.js

```diff
@@ -212,6 +213,9 @@ function logout(bounce) {
 
     // Redirect to logout
     if (bounce) {
+        cookie.set('cs_loggedOut', 'true', {
+            expires: 1 / 11520
```

Comment:
Can we add this to constant? Is this a 5s right?

Reply:
As per above, we should change this to 8s

Reply:
done